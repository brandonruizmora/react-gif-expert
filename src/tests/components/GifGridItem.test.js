import GifGridItem from '../../components/GifGridItem';
import { shallow } from 'enzyme';
import '../../setupTests';

describe('Pruebas en <GifGridItem />', () => {
    
    test('Debe renderizar GifGridItem correctamente', () => {
        const data = {title: 'Magos', url: 'https://gooogle.com'};
        const wrapper = shallow(<GifGridItem {...data} />);
        expect(wrapper).toMatchSnapshot();
    });
    
});
