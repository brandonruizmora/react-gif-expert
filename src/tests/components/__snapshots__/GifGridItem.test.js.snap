// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> Debe renderizar GifGridItem correctamente 1`] = `
<div
  className="card animate__animated animate__jackInTheBox"
>
  <p>
    Magos
  </p>
  <img
    alt="Magos"
    src="https://gooogle.com"
  />
</div>
`;
